growth/too/views.py

```diff
@@ -21,6 +21,7 @@
     as plot_observability
 import matplotlib.style
 import pkg_resources
+from prettytable import PrettyTable
```

Comment:
I don't think that we have a module by this name, do we? Is this adding some dependency?

Reply:
Right, we need to pip install it.  Which other files should be modified?

Reply:
prettytable added to requirements.txt. Astropy.table could be used but needs some more working.

Reply:
See "Text table designed for humans and test having position line before the header line" example at https://docs.astropy.org/en/stable/io/ascii/fixed_width_gallery.html#fixedwidthtwoline.